patches/nuget-client/0007-build-for-.NET-5.patch

```diff
@@ -0,0 +1,67 @@
+From a158ba6c9bdf7465264a5de8ee62e5d3572b961a Mon Sep 17 00:00:00 2001
+From: Tom Deseyn <tom.deseyn@gmail.com>
+Date: Thu, 12 Nov 2020 15:51:37 +0100
+Subject: [PATCH] nuget.client: build for .NET 5
+
+---
+ build/common.project.props                              | 4 ++--
+ build/packages.targets                                  | 4 ++--
+ src/NuGet.Core/NuGet.Frameworks/NuGet.Frameworks.csproj | 2 +-
+ 3 files changed, 5 insertions(+), 5 deletions(-)
+
+diff --git a/build/common.project.props b/build/common.project.props
+index 03c93ae65..4679fd57a 100644
+--- a/build/common.project.props
++++ b/build/common.project.props
+@@ -13,11 +13,11 @@
+     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+     <NETFXTargetFrameworkVersion>v4.7.2</NETFXTargetFrameworkVersion>
+     <NETFXTargetFramework>net472</NETFXTargetFramework>
+-    <NETCoreTargetFramework>netcoreapp2.1</NETCoreTargetFramework>
++    <NETCoreTargetFramework>netcoreapp5.0</NETCoreTargetFramework>
+     <NETCoreTargetFrameworksForSigning>$(NETCoreTargetFramework);netcoreapp5.0</NETCoreTargetFrameworksForSigning>
+     <IsBuildOnlyXPLATProjects>$(DotNetBuildFromSource)</IsBuildOnlyXPLATProjects>
+     <NETCoreTestTargetFramework>netcoreapp5.0</NETCoreTestTargetFramework>
+-    <NetStandardVersion>netstandard2.0</NetStandardVersion>
++    <NetStandardVersion>netcoreapp5.0</NetStandardVersion>
+     <TargetFrameworksExe>$(NETFXTargetFramework);$(NETCoreTargetFramework)</TargetFrameworksExe>
+     <TargetFrameworksExe Condition="'$(IsBuildOnlyXPLATProjects)' == 'true'">$(NETCoreTargetFramework)</TargetFrameworksExe>
+     <TargetFrameworksExeForSigning>$(TargetFrameworksExe);netcoreapp5.0</TargetFrameworksExeForSigning>
+diff --git a/build/packages.targets b/build/packages.targets
+index 873850ccb..baa369e09 100644
+--- a/build/packages.targets
++++ b/build/packages.targets
+@@ -8,7 +8,7 @@
+         <VSServicesVersion>16.153.0</VSServicesVersion>
+         <VSThreadingVersion>16.7.56</VSThreadingVersion>
+         <!-- TODO - remove this when temporary patching is no longer necessary. See https://github.com/nuget/home/issues/8952 -->
+-        <PatchedSystemPackagesVersion>5.0.0-preview.3.20214.6</PatchedSystemPackagesVersion>
++        <PatchedSystemPackagesVersion>5.0.0</PatchedSystemPackagesVersion>
```

Comment:
@dseefeld this change is for getting rid of the System.Security.Cryptography.Cng,5.0.0-preview.3.20214.6 and System.Security.Cryptography.Pkcs,5.0.0-preview.3.20214.6 prebuilts.

Reply:
Cool.  Thanks.  I updated the prebuilt wiki to include the PR and assigned you.